src/domain/challenge/challenge/dto/challenge.dto.create.ts

```diff
@@ -11,4 +11,7 @@ export class CreateChallengeInput extends CreateBaseChallengeInput {
   })
   @IsOptional()
   leadOrganizations?: string[];
+
+  @Field(() => UUID, { nullable: false })
+  lifecycleID!: string;
```

Comment:
Should this be mandatory?

Reply:
Not sure. I asked Neil. It looks as it can be optional.

Reply:
This must be `lifecycleTemplateId`